### DISCUSSION TITLE
Is there a way to render the breadcrumbs above the heading?

### DISCUSSION THREAD

**Discussion by iainhallam** [combostrap/combo#20] [Q&A] [answered]
At present, Combo renders the heading, first section, then breadcrumbs, and I'd like to put them above the page completely so they are above the heading; is that possible?

**Comment by gerardnico** [comment, marked as answer]
Good point. We are working on it actually to finish the modularity system. The renderer should die.

What is in the pipeline:
- we are adding a header and footer slot to the main page.
- the [hiearchical breadcrumb](https://combostrap.com/navigation/combostrap-ui-hierarchical-breadcrumb-q6kfagq5) will become a syntax component

This way, if you create a page called `slot_main_header` with the `breadcrumb` component first, you should get the result that you want.

**Reply by iainhallam** [reply]
That sounds great. Do you have a rough ETA for this work? Thanks.

**Reply by gerardnico** [reply]
Next release over a week or two. 

It seems easy but their is a lot of side effect (heading section editing, unique cache by slot and requested page, ...)

**Comment by gerardnico** [comment]
This is available in the last release:
https://combostrap.com/page/main-header-slot-l49u26nj
